lesson01/index.html

```diff
@@ -7,8 +7,8 @@
   <title>Document</title>
 </head>
 <body>
-  <ul>
+  <ul id = "target">
```

Comment:
IMO
I recommend adding the prefix `js-` to anything involving JavaScript.
[reference article](https://zenn.dev/wagashi_osushi/books/94efd21a66ccaa/viewer/349ad3#javascript%E9%96%A2%E9%80%A3%E3%81%AE%E5%87%A6%E7%90%86)
```suggestion
  <ul id = "js-target">
```

Reply:
Thank you mai-san！
it is easy to recognize.

Reply:
[fixed commit: ](https://github.com/groovywave/JS_Practice/commit/704e11bfa15db8127c273c1fb7913dcd3c8f8d87)